api/api/migrations/0071_alter_audio_options_alter_deletedaudio_options_and_more.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.11 on 2024-09-25 16:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0070_throttledapplication_allowed_origins_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='audio',
+            options={'ordering': ['-created_on'], 'verbose_name': 'audio track', 'verbose_name_plural': 'audio tracks'},
```

Comment:
I wonder what introduced this ordering change? :thinking: And why it wasn't present before. Image has had it since migration 0.